lib/archethic_web/live/dashboard_live.ex

```diff
@@ -11,7 +11,11 @@ defmodule ArchethicWeb.DashboardLive do
       Archethic.Metrics.Poller.monitor()
     end
 
-    {:ok, socket}
+    version = Application.spec(:archethic, :vsn)
+    applications = Application.spec(:archethic, :applications)
+    new_socket = socket |> assign(%{version: version, applications: applications})
```

Comment:
Why do you want to use `applications` ?

Reply:
Was testing something, cleaned this one since not required.